CODEOWNERS

```diff
@@ -22,7 +22,7 @@
 /arch/arm/core/aarch64/                   @carlocaione
 /arch/arm/include/aarch32/cortex_m/cmse.h @ioannisg
 /arch/arm/include/aarch64/                @carlocaione
-/arch/arm/core/aarch32/cortex_r/          @MaureenHelm @galak @ioannisg @bbolen @stephanosio
+/arch/arm/core/aarch32/cortex_ar/         @MaureenHelm @galak @ioannisg @bbolen @stephanosio
```

Comment:
Should it be `cortex_a_r` instead?

Reply:
`cortex_a_r` looked like too many underscores for such a short name, so I went with `cortex_ar` instead; IMHO, the latter looks cleaner and its meaning is clear enough.

What do you think? @ioannisg @andrewboie @galak @MaureenHelm

Reply:
cortex_a_r is cleaner, for sure, but longer; but perhaps not too long, to be used.
No strong opinion, but let's get more opinions, because once we set it up we will have to stay with it for some time.

Reply:
No strong opinion either. `cortex_a_r` is certainly easier to understand and less prone to confusion (is "ar" an architecture?) but aesthetically less pleasing. I will approve this PR but would like others to weigh in as well.

Reply:
I agree with @carlescufi